src/components/dashjs-api-control-element/dashjs-api-control-element.css

```diff
@@ -1,3 +1,14 @@
 :host {
   display: block;
 }
+
+.middle {
```

Comment:
you could also use 
https://github.com/AWTPJ2021/dash-reference-ui/blob/06a1280a0914a031457c52e719bead075c12a9cc/src/components/dashjs-settings-control-element/dashjs-settings-control-element.css#L7-L10

Reply:
alright